app/models/study_file.rb

```diff
@@ -45,6 +45,13 @@ class StudyFile
       'taxon_id' => Taxon.all.map {|t| [t.common_name, t.id.to_s]}
   }
 
+  # map of 'options.[key]' for handling study file bundles, via upload/sync pages
+  BUNDLE_KEY_OPTS = {
+      'MM Coordinate Matrix' => 'matrix_id',
+      'BAM' => 'bam_id',
+      'Cluster' => 'cluster_file_id'
+  }
```

Comment:
It actually originally did but only moved here because of the weird `NoMethodError` issue I was having because of the typo.  I can move it back and rename.

Reply:
b46fb50